packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/technical-details/technical-details.html

```diff
@@ -2,6 +2,14 @@
     class="h-100"
     heading="{{:: 'dedicated_server_dashboard_technical_details_heading' | translate }}"
 >
+    <oui-message
+        data-ng-if="$ctrl.upgradeTask && !$ctrl.upgradeTask.isTaskInitialized()"
```

Comment:
I wonder If we can use the optional chaining here ?
Something like
```suggestion
        data-ng-if="!$ctrl.upgradeTask?.isTaskInitialized()"
```